frontend/src/features/projects/dispose/ProjectDisposeLayout.tsx

```diff
@@ -110,22 +114,11 @@ const ProjectDisposeLayout = ({ match, location }: { match: Match; location: Loc
 
   //If the current route isn't set, set based on the query project status.
   useEffect(() => {
-    if (location.pathname === '/dispose') {
-      if (project.status?.route !== undefined && projectNumber !== undefined) {
-        historyReplace(`/dispose${project.status?.route}?projectNumber=${project.projectNumber}`);
-      } else {
-        dispatch(clearProject());
-      }
+    if (location.pathname === '/dispose' && workflowStatuses?.length > 0) {
```

Comment:
Due to to the previous change in routing behaviour I was able to simplify this further.